cmd/networkspec/listBackups.go

```diff
@@ -0,0 +1,99 @@
+package networkspec
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/OnFinality-io/onf-cli/pkg/printer"
+	"github.com/OnFinality-io/onf-cli/pkg/service"
+	"github.com/spf13/cobra"
+)
+
+func listBackupsCmd() *cobra.Command {
+	printFlags := printer.NewPrintFlags()
+	c := &cobra.Command{
+		Use:   "list-backups",
+		Short: "List all the networks that supports lightning restore/backups",
+		Run: func(cmd *cobra.Command, args []string) {
+			// get public network specs data
+			specs, err := service.GetBackupNetworkSpecs()
+			if err != nil {
+				fmt.Println(err.Error())
+				return
+			}
+
+			// get backups supported public networks data
+			backups, err2 := service.GetBackups()
```

Comment:
The err2 can use err above

Reply:
Oh, right :) Still not used to var declarations in golang. I'll update the code. Thanks!